module/data/spell/__core.json

```diff
@@ -136,6 +490,32 @@
 				}
 			}
 		},
+		{
+			"name": "Sanctuary",
+			"source": "PHB",
+			"_TODO": [
```

Comment:
kinda meh, but the thinking is that we might want a toggle for "give me everything which is 100% done only" vs. "I'll take whatever I can get," and keeping the "TODOs" in with the raw data would allow that to be easily built later (also because JSON doesn't have comments lol)

as far as shipping junk goes, my original stance was "perfected only," but in retrospect that's never going to work.  Users (myself included) are going to want whatever time-savings they can get

Reply:
This is true, and a reason I've been slow on adding my converted stuff here... realising that actually there are edge cases and I don't know how to (yet) handle them. I think sorting #26 is gonna be a roadblock in that respect. Many other things require messy macros and I'm still getting used to writing those 🥲

I'm fine with the `_TODO` to centralise stuff here, at the very least. The only issue is whether `_TODO` is, as with Sanctuary, "the rest of this hasn't been coded yet" (no detrimental impact on game) or actually "there are edge cases that you will need to manually override" (actual detrimental impact on game). I've got a lot of things in the latter category, but at least with the former we can spam out all the 'expires at the end of your next turn' etc. _durations_, even if there's no actual automation of their effects.

Reply:
🤔 could always split it into e.g. `_TODO_EDGE`, `_TODO_ENHANCEMENT`, `_TODO_INCOMPLETE`, ... (or have it all in `_TODO` but be structured as `{"type": "edge", "text": "At midnight, the user must spin backwards three times or remove this effect"}`)

the user could then be allowed to choose whether or not they want to include "incomplete" effects and "has known edge case" effects in the module's config

unsure if there's value to this much granularity, since you're not going to know which effect is suffering from which issue(s) on an effect-by-effect basis, so you wouldn't be able to manually work around it as a GM without e.g. noticing something being broken during play

Reply:
I would personally only split it as `_TODO_EDGE` and `_TODO_INCOMPLETE` as I mentioned above, since what matters to me is whether or not I need to _work against_ the automation or not. For instance, let's say I've automated the Bear's Endurance effect of the _enhance ability_ spell perfectly. That's very nice, but once I start wanting to use the Cat's Grace effect instead, it's not simply a matter of applying the effects manually—I have to also manually disable the Bear's Endurance effect too. While it's just a minor 'disable automatically applied AE' in this specific example, I'm sure there's countless others with more complicated edge cases.

That said, I'm fine with shunting it off as the user's problem if they _choose_ to import incomplete effects... so yeah I actually think just leave it as a pain ` _TODO`  with our dev notes; the exact nature of the missing automation can be left as an exercise for the user 😎